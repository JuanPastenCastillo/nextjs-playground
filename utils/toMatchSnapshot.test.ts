import { expect, test } from "vitest"

test("matches snapshot strict", () => {
  const data = { foo: new Set(["bar", "snapshot"]) }
  expect(data).toMatchSnapshot()
})

test("matches snapshot only shape", () => {
  const data = { foo: new Set(["bar", "snapshot"]) }
  expect(data).toMatchSnapshot({ foo: expect.any(Set) })
})
