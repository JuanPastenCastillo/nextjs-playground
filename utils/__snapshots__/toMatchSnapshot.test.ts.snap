// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`matches snapshot only shape 1`] = `
{
  "foo": Any<Set>,
}
`;

exports[`matches snapshot strict 1`] = `
{
  "foo": Set {
    "bar",
    "snapshot",
  },
}
`;
